app/about_us/OurCrew.js

```diff
@@ -0,0 +1,25 @@
+const OurCrew = () => {
```

Comment:
Be cautious!
You're currently not exporting anything here, so when you try to import the component elsewhere, the app will crash.